src/lattice/reciprocal/paths.js

```diff
@@ -1,486 +1,132 @@
-import _ from "underscore";
+import _ from 'underscore';
 
 /**
  * Default kpoing paths according to:
  *  [AFLOW](https://arxiv.org/abs/1004.2974) methodology.
  *  Paths are split in parts for clarity.
  */
 const points = {
-    "CUB": [
-        [
-            "Г",
-            "X",
-            "M",
-            "Г",
-            "R",
-            "X"
-        ],
-        [
-            "M",
-            "R"
-        ]
+    CUB: [
+        ['Г', 'X', 'M', 'Г', 'R', 'X'],
```

Comment:
The array elements were on newlines for a reason: easier to read and spot errors in each, especially for formulas, and we can move/add elements without affecting the rest of them. Can we keep it the way they were?